bot/src/main/java/com/intern/bot/service/MyBot.java

```diff
@@ -0,0 +1,34 @@
+package com.intern.bot.service;
+
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.objects.Update;
+
+@Slf4j
+public class MyBot extends TelegramLongPollingBot {
+
+  private static final Map<String, String> getenv = System.getenv();
+
+  @Override
+  public void onUpdateReceived(Update update) {
+    // We check if the update has a message and the message has text
+    if (update.hasMessage() && update.getMessage().hasText()) {
+      String inputText = update.getMessage().getText();
+      log.info(inputText);
+    }
+  }
+
+  @Override
+  public String getBotUsername() {
+    return "GridInternBot";
+  }
+
+  @Override
+  public String getBotToken() {
+    //TODO HIDDEN KEEP TOKEN?
+    return "1716055129:AAHWA-utqyyw6DVNYB_TGAHDnU2jPNHLV7o";
```

Comment:
Put this value inside environment variable and never expose it. Use `application.properties` file for storing this value and mapping to `@ConfiguraitonProperties` like I do here: https://github.com/Praytic/discord-reactive-stats-bot/blob/master/src/main/java/org/praytic/discord/statsbot/config/properties/BotProperties.java

There is a cool feature in Spring - when you define property in proerty file like this:
```
bot.token=value
```
You can override it by environment variable with name `BOT_TOKEN=overridden`.

Also you can read about Externalised Configuration here: https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config